terraform/aws/Makefile

```diff
@@ -37,6 +41,7 @@ ifneq ($(INSTALL_VERSION),)
 	export TF_VAR_install_version_url ?= $(shell platform=$(TF_VAR_platform); for channel in unstable current stable; do mixlib-install download chef-server --url -c $$channel -a x86_64 -p $$(echo $${platform%-*} | sed 's/rhel/el/') -l $${platform\#\#*-} -v $(INSTALL_VERSION) 2>/dev/null && break; done | head -n1)
 endif
 
+export UPGRADE_VERSION ?= $(INSTALL_VERSION)
```

Comment:
I found defaulting this meant I didn't need to supply `UPGRADE_VERSION` when not running the upgrade scenario

Reply:
That is right!

Reply:
Yeah, the variable names are confusing...especially if you aren't running an upgrade scenario.